src/prisma/prisma.service.spec.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable fp/no-let */
```

Comment:
I'll disable this rule for tests in our community rules.
In the next version we can remove this comment